presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -176,6 +186,55 @@ public int run(List<QueryPair> queries)
         return failed;
     }
 
+    private int getDefultHashPartitionCount()
+    {
+        String url = config.getTestGateway();
+        String username = Optional.ofNullable(config.getTestUsernameOverride()).orElse(DEFAULT_USER_NAME);
+        String password = Optional.ofNullable(config.getTestPasswordOverride()).orElse(null);
+
+        int defaultHashPartitionCount = 0;
+        int numOfActiveNodes = 0;
+        try (ResultSet result = executeQuery(url, username, password, "select count(*) as NUM_OF_WORKERS from system.runtime.nodes where coordinator=false")) {
+            while (result.next()) {
+                numOfActiveNodes = result.getInt("NUM_OF_WORKERS");
+            }
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try (ResultSet result = executeQuery(url, username, password, "show session")) {
+            while (result.next()) {
+                if ((result.getString("Name")).equals(HASH_PARTITION_COUNT)) {
+                    defaultHashPartitionCount = Integer.parseInt(result.getString("Value"));
+                    break;
+                }
+            }
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        return Math.min(defaultHashPartitionCount, numOfActiveNodes);
+    }
+
+    private ResultSet executeQuery(String url, String username, String password, String sql) throws SQLException
```

Comment:
Put the `throws ...` clause in a new line. You can setup a auto formatter in IntelliJ (https://github.com/prestodb/presto#developers). Also it would be better if you run a mvn build with style check before pushing the PR.